helm/policies-common/tests/abs/test_basic_cluster_default.py

```diff
@@ -0,0 +1,61 @@
+import yaml
+from functools import partial
+import time
+import random
+import string
+import ensure
+from textwrap import dedent
+
+import pytest
+from pytest_kube import forward_requests, wait_for_rollout, app_template
+
+import logging
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.smoke
+def test_cluster_policy(kubernetes_cluster) -> None:
```

Comment:
This test basically checks if we set the labels correctly on a clean `cluster` CR.